ml-proto/runtests.py

```diff
@@ -51,7 +51,7 @@ def find_interpreter(path):
 def rebuild_interpreter(path):
   print("// building %s" % path)
   sys.stdout.flush()
-  exitCode = subprocess.call(["ocamlbuild", "-libs", "bigarray, nums, str", "main.native"], cwd=os.path.abspath("src"))
+  exitCode = subprocess.call(["ocamlbuild", "-libs", "bigarray, nums, str", "-Is", "given, spec, host", "host/main.native"], cwd=os.path.abspath("src"))
```

Comment:
ml-proto/README.md also has a copy of this build command; as long as it's still there, would you mind updating it too?